src/index.js

```diff
@@ -63,7 +62,7 @@ class DelegatedContentRouting {
    * @param {object} options
    * @param {number} options.timeout How long the query can take. Defaults to 30 seconds
    * @param {number} options.numProviders How many providers to find, defaults to 20
-   * @returns {AsyncIterable<PeerInfo>}
+   * @returns {AsyncIterable<{ id: PeerId, addrs: Multiaddr[] }>}
```

Comment:
As mentioned in, https://github.com/libp2p/js-libp2p-delegated-peer-routing/pull/25, I think we should make this return `{ id, multiaddrs }` for api consistency.